### PR TITLE
Updated usage Readme

### PR DESCRIPTION
**Pull request by andrew-hu368**
## In this PR

- Updated readme with examples

### Checklist

- [x] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/ducktors/turborepo-remote-cache/pulls) for the same update/change?

- [x] Have you lint your code with `pnpm lint` locally prior to submission?
- [x] Have you added an explanation of what your changes do and why you'd like us to include them?
- [x] Have you written new tests for your core changes, as applicable?
- [x] Have you successfully ran build with `pnpm build` of your changes locally?
- [x] Have you successfully ran tests with `pnpm test` of your changes locally?
- [x] Have you commit using [Conventional Commits](https://github.com/ducktors/turborepo-remote-cache#how-to-commit)?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,83 @@
...
+// const storage = new Storage({
+//  bucket: 'your-gcp-bucket';
+//  projectId: 'your-gcp-project-id';
+//  clientEmail: 'your-gcp-client-email';
+//  privateKey: 'your-gcp-private-key';
+// })
+
+async function main() {
+  // Your readable stream
+  const readable = new Readable();
```
Comment (fox1t) [inline README.md:35]:
```suggestion
  const readable = new Readable.from("Hello, world!");
```

File: README.md [hunk 2]
```diff
@@ -1,24 +1,83 @@
...
+  readable.push("Hello, world!");
```
Comment (fox1t) [inline README.md:36]:
```suggestion
```

File: README.md [hunk 3]
```diff
@@ -1,24 +1,83 @@
...
+  // copy file
+  const copiedFileString = await storage.copy(
+    "your-file.txt",
+    "your-file-copy.txt"
+  );
+
+  // move a file
+  const movedFileString = await storage.move(
+    "your-file-copy.txt",
+    "copied/your-file-copy.txt"
```
Comment (fox1t) [inline README.md:56]:
```suggestion
    "moved/your-file-copy.txt"
```

File: README.md [hunk 4]
```diff
@@ -1,24 +1,83 @@
...
+  );
+
+  // delete a file
+  await storage.remove("your-file.txt", "your-file-copy.txt");
```
Comment (fox1t) [inline README.md:60]: I am not sure I understand this example.

**Review by fox1t — CHANGES_REQUESTED** [review]

File: README.md (hunk 4 above)
Comment (andrew-hu368) [reply README.md]: Removed the second arg

**Review by fox1t — APPROVED** [review]

**Comment by matteovivona** [comment]
@all-contributors please add @andrew-hu368  for doc

**Pull request merged**